src/common/options.cc

```diff
@@ -3631,11 +3631,11 @@ std::vector<Option> get_global_options() {
     .set_description("Time in seconds to sleep before next removal transaction for HDDs"),
 
     Option("osd_delete_sleep_ssd", Option::TYPE_FLOAT, Option::LEVEL_ADVANCED)
-    .set_default(0)
+    .set_default(1)
```

Comment:
@ifed01 could you please update these values in doc/rados/configuration/osd-config-ref.rst as well?